docs/sources/write/front-matter/index.md

```diff
@@ -99,11 +99,11 @@ To determine the relative alias, you must first understand the meaning of the cu
 
 For an alias in the page `/docs/grafana/latest/alerting/manage-notifications/`:
 
-- The current directory element (`.`) refers to the directory containing the current page, _not_ the directory of the current page.
+- The current directory element (`.`) refers to the directory that contains the current page's directory
```

Comment:
What do you think about keeping the `_not_...`? I've noticed a lot of people expect the `.` to mean the current directory like in shells like Bash.


```suggestion
- The current directory element (`.`) refers to the directory that contains the current page's directory, _not_ the directory of the current page.
```

Reply:
Accepted the suggestion, thank you! 

Should the zizmor check on this be failing?